DemoApp/appcenter-post-clone.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 echo "Executing post clone script in `pwd`"
 echo $NPM_RC | base64 --decode > $APPCENTER_SOURCE_DIRECTORY/DemoApp/.npmrc
+echo $GOOGLE_SERVICES_JSON | base64 --decode > $APPCENTER_SOURCE_DIRECTORY/DemoApp/android/app/google-services.json
```

Comment:
We decided to commit google-services.json but removing secrets. You can look at how we did it in Android and use the same format (make sure it contains only the minimum information and empty API keys). That way we can remove the App Center build variable and also not have to worry when we clone the SDK.